### PR TITLE
Update chaos (Fixes #166)

### LINKED ISSUE #166: Unify Chaos
Adding linkages for marked Slaves/Everchosen units to be included god aligned armies.

Also going to check assigned phase events to look for areas to clean up the amount of text displayed.

### PR DESCRIPTION
**Pull request by JohnSchramm**
Ended up updated the other armies I submitted too since they all needed some text reduction.

### CODE REVIEW

File: src/army/khorne/units.ts [hunk 1]
```diff
@@ -17,9 +17,41 @@ import {
   START_OF_MOVEMENT_PHASE,
   START_OF_TURN,
 } from 'types/phases'
+import SlavesToDarkness from 'army/slaves_to_darkness'
+import Everchosen from 'army/everchosen'
+
+// Importing Nurgle markable Slaves to Darkness units.
+const getSlavesUnits = () => {
```
Comment (daviseford) [inline src/army/khorne/units.ts]: Awesome! I'm so glad to have people that know all the intricacies of these armies!

File: src/army/khorne/units.ts [hunk 2]
```diff
@@ -17,9 +17,41 @@ import {
...
+    `Chaos Knights`,
+    `Chaos Marauder Horsemen`,
+    `Chaos War Mammoth`,
+  ]
+  return SlavesToDarkness.Units.filter(unit => listOfUnits.includes(unit.name))
+}
+
+// Importing god aligned Everchosen units.
+const getEverchosenUnits = () => {
+  const listOfUnits = [`Archaon`]
```
Comment (daviseford) [inline src/army/khorne/units.ts]: Cool!

File: src/army/khorne/units.ts [hunk 3]
```diff
@@ -17,9 +17,41 @@ import {
   START_OF_MOVEMENT_PHASE,
   START_OF_TURN,
 } from 'types/phases'
+import SlavesToDarkness from 'army/slaves_to_darkness'
+import Everchosen from 'army/everchosen'
+
+// Importing Nurgle markable Slaves to Darkness units.
```
Comment (daviseford) [inline src/army/khorne/units.ts]: Nurgle in a Khorne file?

File: src/army/slaanesh/units.ts [hunk 4]
```diff
@@ -13,9 +14,43 @@ import {
   START_OF_HERO_PHASE,
   START_OF_GAME,
 } from 'types/phases'
+import SlavesToDarkness from 'army/slaves_to_darkness'
+import Everchosen from 'army/everchosen'
+
+// Importing Nurgle markable Slaves to Darkness units.
```
Comment (daviseford) [inline src/army/slaanesh/units.ts]: Nurgle in Slaanesh file?

File: src/army/tzeentch/units.ts [hunk 5]
```diff
@@ -14,9 +14,43 @@ import {
   START_OF_HERO_PHASE,
   START_OF_SHOOTING_PHASE,
 } from 'types/phases'
+import SlavesToDarkness from 'army/slaves_to_darkness'
+import Everchosen from 'army/everchosen'
+
+// Importing Nurgle markable Slaves to Darkness units.
```
Comment (daviseford) [inline src/army/tzeentch/units.ts]: Nurgle in Tzeentch file?

**Comment by JohnSchramm** [comment]
Ok changed the comments.  I'll try to "slow my roll" next time when doing that, lol!

**Pull request merged**